src/cpp/include/pre-generated/EVENT/TrackState.h

```diff
@@ -27,8 +27,7 @@ class TrackState : public LCObject {
 
 public: 
     /// Destructor.
-    virtual ~TrackState() = default;
-
+    //virtual ~TrackState() { /* nop */; } // implicit!
```

Comment:
```suggestion
    //virtual ~TrackState() = default; // implicit!
```
Personal preference for indicating that this is no-op. On the other hand we could also entirely remove the commented one here, c.f. rule of 0.

Reply:
Apart from the commenting, this change is because I didn't start from the HEAD of master and the line was actually changed, and when I solved the conflict I just took the line from my changes. Yes, completely removing is fine of course

PS: cf. is one word, cf. https://en.wikipedia.org/wiki/Cf. /nitpick

Reply:
Removed the commented line